inst/extdata/temp-data-inject/L201.nonghg_steepness.csv

```diff
@@ -1,7 +1,7 @@
-INPUT_TABLE
-Variable ID
-GDPCtrlSteep
-
+# INPUT_TABLE
+# Variable ID
+# GDPCtrlSteep
+#
```

Comment:
I did not know that a level 2 file could feed back into Level 1--this is just weird.

>should I make edits to this input file 

No. I'm impressed that `get_data` was even able to read it, honestly. Let me check out your branch and take a look at this later.